src/core/Akka.Cluster.Tests.MultiNode/Routing/UseRoleIgnoredSpec.cs

```diff
@@ -272,7 +272,7 @@ private void A_cluster_must_group_local_on_role_b()
                 var router = Sys.ActorOf(
                     new ClusterRouterGroup(
                         new RoundRobinGroup(paths: null),
-                        new ClusterRouterGroupSettings(6, ImmutableHashSet.Create("/user/foo", "/user/bar"), allowLocalRoutees: false, useRole: role)).Props(),
+                        new ClusterRouterGroupSettings(6, ImmutableHashSet.Create("/user/foo", "/user/bar"), allowLocalRoutees: true, useRole: role)).Props(),
```

Comment:
As it turns out, this is a bug. `allowLocalRoutees` is supposed to be set for `true` for this part of the spec, per the JVM implementation for `router-3b`:

https://github.com/akka/akka/blob/a94f7cdc98332875e21900017c7eac52dbfa2bbd/akka-cluster/src/multi-jvm/scala/akka/cluster/routing/UseRoleIgnoredSpec.scala#L193

```scala
          ClusterRouterGroupSettings(totalInstances = 6, routeesPaths = List("/user/foo", "/user/bar"),
            allowLocalRoutees = true, useRole = role)).props,
          "router-3b")
```

This is why this spec didn't catch the bug earlier. The two bugs cancelled eachother out.